tox.ini

```diff
@@ -32,42 +32,12 @@ commands =
     isort {[vars]all_path}
     black {[vars]all_path}
 
-[testenv:lint]
```

Comment:
> Rip out the linters since they don't play nicely with integration tests

Can you elaborate?
The static for example saved me some time in the past with async mistakes.

Reply:
```
static run-test-pre: PYTHONHASHSEED='2332775060'
static run-test: commands[0] | mypy /home/rbarry/charms/cos-lite-bundle/tests/
tests/integration/helpers.py:11: error: Skipping analyzing "pytest_operator.plugin": module is installed, but missing
library stubs or py.typed marker  [import]
    from pytest_operator.plugin import OpsTest
    ^
tests/integration/helpers.py:11: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
tests/integration/test_bundle.py:24: error: Skipping analyzing "juju": module is installed, but missing library stubs
or py.typed marker  [import]
    import juju
    ^
tests/integration/test_bundle.py:33: error: Skipping analyzing "pytest_operator.plugin": module is installed, but
missing library stubs or py.typed marker  [import]
    from pytest_operator.plugin import OpsTest
    ^
Found 3 errors in 2 files (checked 3 source files)
ERROR: InvocationError for command /home/rbarry/charms/cos-lite-bundle/.tox/static/bin/mypy tests (exited with code 1)
```

And that's a start (I'd certainly expect more `mypy` errors once these are fixed, since it fails fast).

Almost all of our repos have these disabled for the integration tests for a variety of reasons (`mypy` not being happy with test charms, missing stubs, etc), and the return on investment in keeping an analysis tool happy for code which doesn't ship, or, in the case of this repo, literally has no Python code at all which is _not_ the tests compared to "do the tests execute properly" isn't there, at least fro this repo.